### PR TITLE
fix lint issues

### PR DESCRIPTION
**Pull request by stelar7**

### CODE REVIEW

**Comment by Querijn** [comment]
Merge issues

File: data/message.markdown [hunk 1]
```diff
@@ -4,8 +4,8 @@ Here's some tips to help you on your way:
 1. There's a good chance the AnswerHub already has an answer for you. Be sure to search there, it will help you much faster. https://discussion.developer.riotgames.com/
 2. If you have any questions about the Riot API, I recommend you post them in #riot-api. Please don't ask if you can ask a question, just ask it! If someone knows the answer, they will respond, while "Can I ask a question?" will rarely be replied to.
 3. The pins in the Riot API channel might hold some resources that are not on DDragon. If they're not in the pins, be sure to check out CommunityDragon; http://stelar7.no/cdragon/
-4. Heimerdinger is a bot in the server that will help you with the API. Just paste any Riot API url and it will give you the results that it should return. Make sure that your key is not in the URL! Heimerdinger will add his own. 
-5. Heimerdinger can tell you the status of the API with `!status`.
+4. I can help you find issues in your url, and also show you the results. Just paste any Riot API url and I will work my magic. Make sure that your key is not in the URL!
```
Comment (Querijn) [inline data/message.markdown]: This isnt a lint issue

File: package-lock.json [hunk 2]
```diff
@@ -1,1233 +0,0 @@
-{
```
Comment (Querijn) [inline package-lock.json]: why is this file removed

File: data/message.markdown (hunk 1 above)
Comment (stelar7) [reply data/message.markdown]: i cant find that file in my commits..??

File: package-lock.json (hunk 2 above)
Comment (stelar7) [reply package-lock.json]: same as above
